portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -28,6 +28,10 @@
 
 @WebServlet("/login-status")
 public class LoginStatusServlet extends HttpServlet {
+  
+  private static final Gson gson = new Gson();
+  private static final String urlToRedirectToAfterUserLogsOut = "/";
```

Comment:
Please follow the constant names convention. For example: `URL_TO_REDIRECT_TO_AFTER_USER_LOG_OUT`
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Added this change to PR #18